sched/scheduler/cluster_state.go

```diff
@@ -221,6 +230,7 @@ func (c *clusterState) update(updates []cluster.NodeUpdate) {
 					newNode.readyCh = nil
 					log.Infof("Added new node: %v (%#v), now have %d healthy nodes (%d suspended)",
 						update.Id, update.Node, len(c.nodes), len(c.suspendedNodes))
+					c.numIdle++
```

Comment:
I believe so, if it's suspended it's offline so bringing it back puts it in the idle pool again. There are other issues with numIdle that I've already fixed but haven't pushed.